internal/services/postgres/postgresql_server_resource.go

```diff
@@ -76,7 +76,10 @@ func resourcePostgreSQLServer() *pluginsdk.Resource {
 				return []*pluginsdk.ResourceData{d}, err
 			}
 
-			resp, err := client.Get(ctx, *id)
+			timeout, cancel := context.WithTimeout(ctx, 5*time.Minute)
```

Comment:
Can we use the `read` timeout here instead?
```suggestion
			timeout, cancel := context.WithTimeout(ctx,, d.Timeout(pluginsdk.TimeoutRead))
```

Reply:
@jackofallops thanks for your feedback. Code has been updated. Could you please take another look?